wow_message_parser/src/doc_printer/container.rs

```diff
@@ -434,8 +434,26 @@ fn print_container_examples(s: &mut DocWriter, e: &Container, o: &Objects) {
 
         let mut values = HashMap::new();
 
-        for m in e.members() {
-            print_container_example_member(s, e, m, &mut bytes, &mut values, o, e.tags(), "");
+        if e.tags().compressed() {
+            // All fully compressed messages have a u32 at the start with the decompressed size.
+            let _decompressed_size: Vec<&u8> = bytes.clone().take(4).collect();
```

Comment:
Nope, looks like I completely forgot about that. Should it be printed at the start of the example? I can't see an easy way of just printing an additional custom member that isn't already part of the spec without duplicating code.

This sort of ties into your point about the examples being unclear. Arguably the examples shouldn't really include the uncompressed data at all. We don't really care about what the uncompressed raw data looks like (as it will be represented via a Rust struct anyway).

Reply:
AFAIK the compressed messages all exist in uncompressed formats already, so perhaps the solution is to link to the uncompressed equivalent (COMPRESSED_UPDATE_OBJECT -> UPDATE_OBJECT, COMPRESSED_MOVES -> MONSTER_MOVE) and only show the compressed data on the actual page

Reply:
That's an interesting idea and very doable, but I don't want to lock ourselves in to a specific way of handling compressed messages in case a new message breaks this pattern. For now I think it'll be good enough if you don't uncompress and parse the example, but instead just parse the size u32 and add a comment and then parse the rest of the message.

Then manually in the `comment` tag write something like `Compressed version of [SMSG_UPDATE_OBJECT]. Has the same fields when uncompressed`.

Like so

```
0, 0, 0, 0, // Uncompresed size
0, 0, 0, [..],
0, 0, 0, 0, // Compressed data
```